vignettes/philosophy.Rmd

```diff
@@ -0,0 +1,163 @@
+---
+title: "Philosophy of the checklist package"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Philosophy of the checklist package}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+references:
+- type: article-journal
+  id: Baath2012
+  author:
+  - family: Bååth
+    given: Rasmus
+  issued:
+    date-parts:
+    - - 2012
+      - 12
+  title: The State of Naming Conventions in R
+  container-title: The R Journal
+  volume: 4
+  number: 2
+  page: 74--75
+  doi: 10.32614/RJ-2012-018
+  URL: https://doi.org/10.32614/RJ-2012-018
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Everybody develops their own coding habits and style.
+Some people take a lot of effort in making their source code [readable](https://en.wikipedia.org/wiki/Computer_programming#Readability_of_source_code), while others don't bother at all.
+Working together with different people is easier when everyone uses the same standard.
+The `checklist` package defines a set of standards and provides tools to valid whether your project or R package adheres to these standards.
+You can run these tools interactive on your machine.
+You can also add this checks as [GitHub actions](https://github.com/features/actions), which runs then checks automatically after every [push](https://github.com/git-guides/git-push) to the repository on [GitHub](https://github.com).
```

Comment:
Do you add checks **AS** GitHub actions, or do you add checks **IN** GitHub actions?  Sounds a bit strange to me, maybe reword it completely, e.g. 'Add GitHub actions with the checks included'.